src/pk/asn1/der/sequence/der_decode_subject_public_key_info.c

```diff
@@ -40,6 +40,15 @@ int der_decode_subject_public_key_info(const unsigned char *in, unsigned long in
         unsigned int algorithm, void* public_key, unsigned long* public_key_len,
         unsigned long parameters_type, ltc_asn1_list* parameters, unsigned long parameters_len)
 {
+   return der_decode_subject_public_key_info_ex(in, inlen, algorithm, public_key, public_key_len,
+                                                parameters_type, parameters, parameters_len, NULL);
+}
+
+int der_decode_subject_public_key_info_ex(const unsigned char *in, unsigned long inlen,
```

Comment:
this is a "private" function, just change the default signature and adapt all its invocations